.github/workflows/ci_linux_arm64.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - Linux arm64
+
+on:
+  schedule:
+    # Weekdays at 13:00 UTC = 05:00 PST / 06:00 PDT.
+    - cron: "5 4 * * 1-5"
+  workflow_dispatch:
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit). The workflow name is prepended to avoid conflicts between
+  # different workflows.
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  # TODO: Switch to GitHub-hosted arm64 runners when available?
+  # TODO: Switch runs-on labels to use different (non-GCP) self-hosted runners?
+  build_test_all_arm64:
```

Comment:
can we rename this now to include platform for consistency? (build_test_all_linux_arm64, for whenever we have a build_test_all_<mac/windows/etc>)

Reply:
Renamed. Also dropped the `build_test_all_` prefix.